docs/storage-drivers/hdfs.md

```diff
@@ -0,0 +1,28 @@
+<!--[metadata]>
++++
+title = "HDFS storage driver"
+description = "Explains how to use the HDFS storage drivers"
+keywords = ["registry, service, driver, images, storage, HDFS"]
++++
+<![end-metadata]-->
+
+
+# HDFS storage driver
+
+An implementation of the `storagedriver.StorageDriver` interface which uses Hadoop HDFS for object storage.
```

Comment:
This should probably mention that this is for http://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/WebHDFS.html.